lib/tasks/server/middleware/ember-cli:broccoli:serve-files/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var cleanBaseURL = require('clean-base-url');
+var logger       = require('heimdalljs-logger')('ember-cli:broccoli:serve-files');
+
+/**
+ * This addon is used to set the response headers for the assets that will be
+ * served by the next middleware. It waits for the watcher promise to resolve before
+ * setting the response headers.
```

Comment:
Is this comment right?  it looks like response headers are actually set [in the watcher](https://github.com/ember-cli/broccoli-middleware/blob/aac8215f5184b226c51e964313c5c49460cf0da1/lib/watcher-middleware.js#L32)